From b4125c8851cf126ed54d57cbd8e9c1a541e94862 Mon Sep 17 00:00:00 2001
From: Charles Stevens <charles.stevens@compasses.com>
Date: Thu, 22 Jul 2021 12:07:16 -0500
Subject: [PATCH 51/57] RSOM-257 - Change "rise time" to more closely match
 board

Changing this parameter will change the results of the search for a clock
divider setting and therefore cause the driver to pick a different one
to be below the target clock rate (400 kHz)

Signed-off-by: Charles Stevens <charles.stevens@compasses.com>
---
 arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
index 6192d1e2febd..5185564e9da1 100644
--- a/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
+++ b/arch/arm64/boot/dts/renesas/beacon-renesom-baseboard.dtsi
@@ -351,6 +351,7 @@
 &i2c2 {
 	status = "okay";
 	clock-frequency = <400000>;
+	i2c-scl-rising-time-ns = <50>;
 
 	pinctrl-0 = <&i2c2_pins>;
 	pinctrl-names = "default";
-- 
2.17.1

